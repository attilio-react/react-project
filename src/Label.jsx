import React from 'react';

class Label extends React.PureComponent {
  render () {
    return <div>{this.props.text}</div>;
  }
}


export {Label}


