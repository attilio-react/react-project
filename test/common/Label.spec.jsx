import React from 'react';
import renderer from 'react-test-renderer';
import {Label} from '|common/Label.jsx';

describe('Label (Snapshot)', () => {
  it('Label renders the text', () => {
    const component = renderer.create(<Label text='hello' />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});

