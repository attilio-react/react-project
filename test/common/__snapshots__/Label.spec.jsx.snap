// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label (Snapshot) Label renders the text 1`] = `
<span>
  hello
</span>
`;
